MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -130,6 +134,16 @@ export class ChatsComponent implements OnInit, OnDestroy {
     }).catch(err => console.error(err.toString()));
   }
 
+  onJoinChatClick() : void {
+    this._userChatsService.joinCommunity(this.activeChatId).pipe(takeUntil(this.componentDestroyed$)).subscribe( {
+      next: _ => {
+        this.searchChatQuery = '';
+        this.chatFilter = 'All chats';
+        this.getChats();
```

Comment:
Add signal R hub connection in method of group join so that user will see new messages without page reload